yt_dlp/extractor/mixch.py

```diff
@@ -62,20 +69,44 @@ class MixchArchiveIE(InfoExtractor):
             'id': '421',
             'title': '96NEKO SHOW TIME',
         }
+    }, {
+        'url': 'https://mixch.tv/archive/1213',
+        'skip': 'paid video, no DRM. expires at Dec 31, 2023',
+        'info_dict': {
+            'id': '1213',
+            'title': '【特別トーク番組アーカイブス】Merm4id×燐舞曲 2nd LIVE「VERSUS」',
+            'live_status': 'not_live',
+            'release_timestamp': 1701421200,
```

Comment:
While I can't run the tests myself this seems incomplete. Shouldn't there be a `release_date` and `thumbnail`?

Reply:
Sorry I didn't run the test. I just copied some fields from "info_json" to the tests.